web-api/src/lambdas/v2/v2ApiWrapper.ts

```diff
@@ -11,19 +11,19 @@
  * 500: returned by wrapping any Error which doesn't specify a statusCode.
  */
 const V2_API_ERROR_STATUS_CODES = [401, 403, 404, 500];
+type ErrorWithStatusCode = Error & { statusCode?: number };
```

Comment:
I keep seeing this line. Perhaps we should export this type so it can be reused?